src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIPipelineService.java

```diff
@@ -201,21 +232,42 @@ private String localizedParamOptionLabel(Locale locale, String workflowName, Str
 				.map(wp -> {
 					Map<String, String> inputParameter = wp.getInputParameters();
 
-					// Go through the parameters and see which ones are getting overwritten.
-					List<PipelineParameter> parameters = defaultParameters.stream()
-							.map(parameter -> {
-								if (inputParameter.containsKey(parameter.getName())) {
-									return new PipelineParameter(parameter.getName(), parameter.getLabel(),
-											inputParameter.get(parameter.getName()));
-								}
-								return new PipelineParameter(parameter.getName(), parameter.getLabel(),
-										parameter.getValue());
-							})
+            // Go through the parameters and see which ones are getting overwritten.
+            List<PipelineParameter> parameters = defaultParameters.stream()
+                    .map(parameter -> {
+                        if (inputParameter.containsKey(parameter.getName())) {
+                            return new PipelineParameter(parameter.getName(), parameter.getLabel(),
+                                    inputParameter.get(parameter.getName()));
+                        }
+                        return new PipelineParameter(parameter.getName(), parameter.getLabel(), parameter.getValue());
+                    })
 							.collect(Collectors.toList());
-					return new SavedPipelineParameters(wp.getId(), wp.getLabel(), parameters);
-				})
-				.collect(Collectors.toList()));
+            return new SavedPipelineParameters(wp.getId(), wp.getLabel(), parameters);
+        })
+                .collect(Collectors.toList()));
 
 		return savedParameters;
 	}
+
+    private List<UIReferenceFile> getReferenceFilesForPipeline(List<Project> projects) {
+        return projects.stream()
+                .map(project -> {
+					List<UIReferenceFile> list = new ArrayList<>();
+					for (Join<Project, ReferenceFile> projectReferenceFileJoin : referenceFileService.getReferenceFilesForProject(
+							project)) {
+						try {
+							ReferenceFile file = projectReferenceFileJoin.getObject();
+							Path path = file.getFile();
+							String filesize = FileUtilities.humanReadableByteCount(Files.size(path), true);
+							UIReferenceFile uiReferenceFile = new UIReferenceFile(projectReferenceFileJoin, filesize);
+							list.add(uiReferenceFile);
+						} catch (IOException e) {
+							e.printStackTrace();
```

Comment:
Add a logger to this file and log the stacktrace at the `error` level

Reply:
Added in 3d580f13b80597ef3efffc931916a6a89528f970